grails-app/i18n/messages.properties

```diff
@@ -353,6 +353,9 @@ default.color.label=Color
 default.comment.label=Comment
 default.comment.message=Enter a comment ...
 default.comments.label=Comments
+default.comments.created.label=Comment has been added
+default.comments.updated.label=Comment has been updated
+default.comments.deleted.label=Comment has been deleted
```

Comment:
Again, these messages should just use the default messages i.e. default.updated.message

The messages above (i.e. default.comment.label should follow conventions and look like this
```
comment.label=Comment
comments.label=Comments
comment.comment.placeholder=Enter a comment...
```